### PR TITLE
Ikasan 1735 adding flow control to metadata

### PR DESCRIPTION
**Pull request by mick-stewart73**

### CODE REVIEW

File: ikasaneip/module/src/main/java/org/ikasan/module/IkasanModuleAutoConfiguration.java [hunk 1]
```diff
@@ -58,6 +58,7 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.DependsOn;
+import org.springframework.context.annotation.Lazy;
```
Comment (andrzej-majewski) [inline ikasaneip/module/src/main/java/org/ikasan/module/IkasanModuleAutoConfiguration.java]: This annotation is not used, so actually is there any change to this class comparing to previous version?

File: ikasaneip/module/src/main/java/org/ikasan/module/service/ModuleServiceImpl.java [hunk 2]
```diff
@@ -44,6 +44,7 @@
 import org.slf4j.Logger; import org.slf4j.LoggerFactory;
 import org.ikasan.module.startup.StartupControlImpl;
 import org.ikasan.module.startup.dao.StartupControlDao;
+import org.ikasan.spec.dashboard.DashboardRestService;
```
Comment (andrzej-majewski) [inline ikasaneip/module/src/main/java/org/ikasan/module/service/ModuleServiceImpl.java]: DashboardRestService  does not seems to be in use.

so there should be no change in the class at all?

File: ikasaneip/module/src/test/java/org/ikasan/module/service/ModuleServiceImplTest.java [hunk 3]
```diff
@@ -70,6 +71,7 @@
     ModuleContainer moduleContainer = mockery.mock(ModuleContainer.class);
     SystemEventService systemEventService = mockery.mock(SystemEventService.class);
     StartupControlDao startupControlDao = mockery.mock(StartupControlDao.class);
+    DashboardRestService moduleMetadataDashboardRestService = mockery.mock(DashboardRestService.class);
```
Comment (andrzej-majewski) [inline ikasaneip/module/src/test/java/org/ikasan/module/service/ModuleServiceImplTest.java]: Not in use?

File: ikasaneip/module/src/main/java/org/ikasan/module/IkasanModuleAutoConfiguration.java (hunk 1 above)
Comment (mick-stewart73) [reply ikasaneip/module/src/main/java/org/ikasan/module/IkasanModuleAutoConfiguration.java]: @andrzej-majewski sorted

File: ikasaneip/module/src/main/java/org/ikasan/module/service/ModuleServiceImpl.java (hunk 2 above)
Comment (mick-stewart73) [reply ikasaneip/module/src/main/java/org/ikasan/module/service/ModuleServiceImpl.java]: @andrzej-majewski sorted

File: ikasaneip/module/src/test/java/org/ikasan/module/service/ModuleServiceImplTest.java (hunk 3 above)
Comment (mick-stewart73) [reply ikasaneip/module/src/test/java/org/ikasan/module/service/ModuleServiceImplTest.java]: @andrzej-majewski sorted

**Review by andrzej-majewski — APPROVED** [review]

**Pull request merged**